src/routes/api/urls.go

```diff
@@ -22,6 +24,7 @@ func UrlsRoute() func(router fiber.Router) {
 
 	return func(router fiber.Router) {
 		router.Get("/", getAllUrls)
+		router.Get("/admin", security.MandatoryAdminApiKeyAuthMiddleware, getAllUrlsAdmin)
```

Comment:
i was preferring not to have `/admin/urls` or `/urls/admin` separately but simply use the `/urls` endpoint with the `X-API-Key` to work for admins.

Reply:
Sure I'll do that